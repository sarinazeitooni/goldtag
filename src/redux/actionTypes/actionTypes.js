const actionTypes={
    Rating:"Rating",
    Comments : "Comments",
    Like : "Like",
    Dislike : 'Dislike'
}
export default actionTypes;