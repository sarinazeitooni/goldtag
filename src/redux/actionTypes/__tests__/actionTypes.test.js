import React from 'react';
import { shallow } from 'enzyme';
import actionTypes from "../actionTypes";
describe("actionTypes hook" , ()=>{
    it("it should render",()=>{
        const container = shallow(<actionTypes/>)
        expect(container).toMatchSnapshot();
    });
});