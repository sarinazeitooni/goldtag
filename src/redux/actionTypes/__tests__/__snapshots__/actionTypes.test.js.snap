// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actionTypes hook it should render 1`] = `ShallowWrapper {}`;
